features/support/hooks.js

```diff
@@ -24,6 +26,14 @@ async function resetEngine (sdk, index) {
   });
 }
 
+async function createNodeCollection (sdk) {
```

Comment:
Some of those calls could be made in parallel

Reply:
Done